cmd/serf/command/agent/command.go

```diff
@@ -302,6 +302,43 @@ func (c *Command) setupAgent(config *Config, logOutput io.Writer) *Agent {
 		return nil
 	}
 
+	if config.ProfileOverrides.StreamTimeout != 0 {
+		serfConfig.MemberlistConfig.TCPTimeout = config.ProfileOverrides.StreamTimeout
+	}
+	if config.ProfileOverrides.IndirectChecks != 0 {
+		serfConfig.MemberlistConfig.IndirectChecks = config.ProfileOverrides.IndirectChecks
+	}
+	if config.ProfileOverrides.RetransmitMult != 0 {
+		serfConfig.MemberlistConfig.RetransmitMult = config.ProfileOverrides.RetransmitMult
+	}
+	if config.ProfileOverrides.SuspicionMult != 0 {
+		serfConfig.MemberlistConfig.SuspicionMult = config.ProfileOverrides.SuspicionMult
+	}
+	if config.ProfileOverrides.SuspicionMaxTimeoutMult != 0 {
+		serfConfig.MemberlistConfig.SuspicionMaxTimeoutMult = config.ProfileOverrides.SuspicionMaxTimeoutMult
+	}
+	if config.ProfileOverrides.PushPullInterval != 0 {
```

Comment:
In the docs section we say that setting this to 0 will disable it, but that won't actually happen this way - the check should probably be `config.ProfileOverrides.PushPullIntervalRaw != ""` instead (the same goes for GossipInterval below) both here and in MergeConfig.

Reply:
Good call, I've made a new commit to update these.